src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -11,11 +11,14 @@
     <FileAlignment>512</FileAlignment>
     <DebugSymbols>true</DebugSymbols>
     <PlatformTarget>AnyCPU</PlatformTarget>
-    <DefineConstants>$(DefineConstants)</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)'=='netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>
+    <EnableWinRT Condition="'$(TargetGroup)'=='netcore50aot'">true</EnableWinRT>
```

Comment:
I know we've been including this from the internal project files, but I'm not sure it actually affects anything with the open build targets. To use WinRT API's we just add the WinRT nuget package. Is that right, @weshaggard ?